.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Bazel build
+
+on:
+    push: {}
+    pull_request: {}
+
+jobs:
+    build:
+        name: Build tcmalloc
+        runs-on: ubuntu-latest
+        steps:
+
+        - name: Checkout repository
+          uses: actions/checkout@v2
+
+        - name: Setup bazel
```

Comment:
Drive by question: Is this step needed? I believe `bazel` is already in the path even without this step. What does this step do?

Reply:
I see @ckennelly already asked this. Dropping my comment in favor of his.